### PR TITLE
Update README.md and RELEASE.md

### PR DESCRIPTION
**Pull request by zhu733756**
Signed-off-by: zhu733756 <zhu733756@kubesphere.io>

- update RELEASE.md and README.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -52,14 +59,17 @@ Once installed, the Fluent Bit Operator provides the following features:
 
 ## Overview
 
-Fluent Bit Operator defines five custom resources using CustomResourceDefinition (CRD):
+Fluent Operator is mainly composed of Fluent Bit Operator and Fluentd Operator.
```
Comment (benjaminhuo) [inline README.md:62]: Fluent Operator includes CRDs and controllers for both Fluent Bit and Fluentd.

File: README.md [hunk 2]
```diff
@@ -52,14 +59,17 @@ Once installed, the Fluent Bit Operator provides the following features:
...
+
+### Fluent Bit Operator
```
Comment (benjaminhuo) [inline README.md:64]: Simply call it `Fluent Bit` is ok

File: README.md [hunk 3]
```diff
@@ -52,14 +59,17 @@ Once installed, the Fluent Bit Operator provides the following features:
...
 
+Fluent Bit Operator defines five custom resources using CustomResourceDefinition (CRD):
```
Comment (benjaminhuo) [inline README.md:66]: The following CRDs are defined for Fluent Bit

File: README.md [hunk 4]
```diff
@@ -52,14 +59,17 @@ Once installed, the Fluent Bit Operator provides the following features:
...
-- **`Parser`**: Defines parser config sections.
-- **`Filter`**: Defines filter config sections.
-- **`Output`**: Defines output config sections.
+- **`ClusterFluentBitConfig`**: Select cluster-level input/filter/output plugins and generates the final config into a Secret.
+- **`ClusterInput`**: Defines cluster-level input config sections.
+- **`clusterParser`**: Defines cluster-level parser config sections.
+- **`ClusterFilter`**: Defines cluster-level filter config sections.
+- **`ClusterOutput`**: Defines cluster-level output config sections.
 
 Each **`Input`**, **`Parser`**, **`Filter`**, **`Output`** represents a Fluent Bit config section, which are selected by **`FluentBitConfig`** via label selectors. The operator watches those objects, constructs the final config, and finally creates a Secret to store the config. This secret will be mounted into the Fluent Bit DaemonSet. The entire workflow looks like below:
```
Comment (benjaminhuo) [inline README.md:74]: Should add Cluster prefix to Input, parser, Filter, Output and FluentBitConfig here

File: README.md [hunk 5]
```diff
@@ -69,24 +79,36 @@ To enable fluent-bit to pick up and use the latest config whenever the fluent-bi
 
 ![fluentbit-operator](docs/images/fluentbit-operator.svg)
 
-Now users can use Fluentd to perform more advanced processing on logs received from fluent-bit and then forward logs to more sinks like S3, Kafka, Elasticsearch, etc. The whole workflow could be described as below.
+### Fluentd Operator
+
+Fluentd Operator defines five custom resources using CustomResourceDefinition (CRD):
```
Comment (benjaminhuo) [inline README.md:84]: The following CRDs are defined for Fluentd:

**Comment by FeynmanZhou** [comment]
Hi @Bettygogo2021,

Could you please help to review this PR? Thanks.

File: README.md [hunk 6]
```diff
@@ -151,20 +175,53 @@ It means the FluentBit Operator works properly if you see the above messages, yo
 kubectl delete -f https://raw.githubusercontent.com/kubesphere/fluentbit-operator/master/manifests/quick-start/quick-start.yaml
 ```
 
-### Configure Custom Watch Namespaces
+##### Fluentd quick start
 
-When starting the operator, you can pass an optional set of namespaces to watch for resources in with the `--watch-namespaces` flag. It takes a comma-separated list of namespaces to watch:
+Fluentd can receives logs from Fluent Bit, both needs to enable the forward plugin.
```
Comment (benjaminhuo) [inline README.md:180]: Fluentd in Fluent Operator is used to receive logs from Fluent Bit or other sources like HTTP and Syslog, etc.

File: RELEASE.md [hunk 7]
```diff
@@ -22,68 +22,72 @@ This page describes the release process and the currently planned schedule for u
...
 ```
 
-Optionally, you can use this handy `.gitconfig` alias.
+Manually build and push the container image:
 
-```ini
-[alias]
-  tag-release = "!f() { tag=v${1:-$(cat VERSION)} ; git tag -s ${tag} -m ${tag} && git push origin ${tag}; }; f"
+```bash
+make build-op-amd64 -e FO_IMG=<image of fluent operator>
```
Comment (benjaminhuo) [inline RELEASE.md:72]: We've CI pipeline already to build images automatically, do we still need manually build and push?

File: RELEASE.md [hunk 8]
```diff
@@ -22,68 +22,72 @@ This page describes the release process and the currently planned schedule for u
...
-make build
-```
\ No newline at end of file
+tag="$(< VERSION)"
+git tag -a "${tag}" -m "${tag}"
+git push origin "${tag}"
+```
+Commit all the changes.
+
+Go to https://github.com/fluent/fluent-operator/releases/new, associate the new release with the before pushed tag, paste in changes made to `CHANGELOG.md`, add file `setup.yaml` from `manifests/setup/setup.yaml` and then click "Publish release".
```
Comment (benjaminhuo) [inline RELEASE.md:91]: Finally, create a new release:
- Go to https://github.com/fluent/fluent-operator/releases/new
- Associate the new release with the previously pushed tag
- Add release notes based on `CHANGELOG.md`
- Add file `setup.yaml` from `manifests/setup/setup.yaml` and then click "Publish release".

File: RELEASE.md [hunk 9]
```diff
@@ -22,68 +22,72 @@ This page describes the release process and the currently planned schedule for u
...
+
+For patch releases, submit a pull request to merge back the release branch into the `master` branch.
```
Comment (benjaminhuo) [inline RELEASE.md:93]: =>
For patch releases, cherry-pick the commits from the release branch into the `master` branch.

**Review by benjaminhuo — CHANGES_REQUESTED** [review]

File: RELEASE.md (hunk 7 above)
Comment (zhu733756) [reply RELEASE.md:72]: No need. As a supplement here. Shall we remove it?

File: README.md [hunk 10]
```diff
@@ -16,7 +16,7 @@ Once installed, the Fluent Operator provides the following features:
 - [Fluent Operator](#fluent-operator)
   - [Table of contents](#table-of-contents)
   - [Overview](#overview)
-    - [Fluent Bit Operator](#fluent-bit-operator)
+    - [Fluent Bit](#fluent-bit)
     - [Fluentd Operator](#fluentd-operator)
```
Comment (benjaminhuo) [inline README.md:20]: Fluentd Operator => Fluentd

**Review by benjaminhuo — CHANGES_REQUESTED** [review]

File: RELEASE.md (hunk 7 above)
Comment (zhu733756) [reply RELEASE.md:72]: OK.

**Review by benjaminhuo — APPROVED** [review]

**Pull request merged**